src/lib.rs

```diff
@@ -449,6 +452,21 @@ pub trait Rng {
         impls::fill_bytes_via_u64(self, dest)
     }
 
+    /// Fill `dest` entirely with random data.
+    ///
+    /// This is the only method which allows an RNG to report errors while
+    /// generating random data; other methods either handle the error
+    /// internally or panic. This method is intended to allow error-handling
+    /// when seeding keys or PRNGs from external (true) RNGs.
+    /// 
+    /// Other than error handling, this is identical to [`fill_bytes`], and
+    /// has a default implementation simply wrapping [`fill_bytes`].
+    /// 
+    /// [`fill_bytes`]: trait.Rng.html#method.fill_bytes
+    fn try_fill_bytes(&mut self, dest: &mut [u8]) -> Result<(), Error> {
+        Ok(self.fill_bytes(dest))
+    }
```

Comment:
Just as a test I removed the default implementation, and you did not miss to implement it for any wrappers etc. 👍.

I would like to see the default implementations be removed, but good to wait until all the changes for RNG implementations are merged.